### PR TITLE
Phan unittest 2

### PR DESCRIPTION
**Pull request by hanpeter**
Unit test of `krux_boto.cli`. Depends on #6 and #7.

### CODE REVIEW

**Comment by dcrampton** [comment]
Nothing jumps out at me, but you'll want more eyes on it.

File: krux_boto/util.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #
 
 from __future__ import absolute_import
+import string
```
Comment (plathrop) [inline krux_boto/util.py]: This looks incorrect. a) it's unusual to import string anymore and b) I don't see you adding usage of it anywhere.

File: test/boto_test.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ class BotoTest(unittest.TestCase):
 
     def _get_parser(self, args=[]):
         # Get the argparse namespace object with the given args
-        parser = krux.cli.get_parser()
+        parser = get_parser()
```
Comment (plathrop) [inline test/boto_test.py]: -.5 to this.

importing into the local namespace seems convenient until you start working on bigger projects and start wondering "Which get_parser am I calling? The one from library Foo or library Bar?" Now you have to scroll all the way back to the imports to answer that question.

I prefer being explicit about where the call comes from.

File: test/cli_test.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#
+
+from krux.cli import get_parser
+from krux.logging import get_logger
+from krux.stats import get_stats, DummyStatsClient
+from krux_boto.boto import Boto, add_boto_cli_arguments, NAME
+from krux_boto.cli import Application, main
+
+
+class CliTest(unittest.TestCase):
```
Comment (plathrop) [inline test/cli_test.py]: `CLITest` not `CliTest`

File: test/util_test.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+class UtilTest(unittest.TestCase):
+
+    def test_get_instance_region_success(self):
+        """
+        get_instance_region successfully return the region
+        """
+        # Mocking the logger to check for calls later
+        mock_logger = MagicMock(spec=Logger, autospec=True)
+        # Mocking the metadata returned by boto
+        mockock_metadata = {
```
Comment (plathrop) [inline test/util_test.py]: looks like maybe a typo? "mockock"?

**Comment by plathrop** [comment]
No objections that should block merge.

File: krux_boto/util.py (hunk 1 above)
Comment (hanpeter) [inline krux_boto/util.py]: This is to support the `string.lowercase` in the return statement.

**Pull request merged**